CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - `signalfxexporter`: Add validation for `sending_queue` setting (#8026)
 - `internal/stanza`: Add support for arbitrary attribute types (#8081)
 - `resourcedetectionprocessor`: Add confighttp.HTTPClientSettings To Resource Detection Config Fixes (#7397)
+- `tanzuobservabilityexporter`: nstrumentation Library and Dropped Counts to Span Tags (#8120)
```

Comment:
```suggestion
- `tanzuobservabilityexporter`: instrumentation Library and Dropped Counts to Span Tags (#8120)
```

Reply:
Hello codeboten, deosu is out-of-office this week, and I am taking over this PR.  I attempted to apply your suggested change to this PR, but I get permission denied errors when I try to push the change.  I am wondering if deosu gave others write access to this PR.  Can you try fixing the typo as well?  Thank you.